src/app/admin/_lib/consts.js

```diff
@@ -0,0 +1,21 @@
+const Resources = {
```

Comment:
🟢 Are you sure we need this object?

Reply:
I though it would be a good idea to store resources names separately. I call "useGetList" a multiple times in different components. Without this obj I had to manually put "therapy" or "specialization". Also, in adminApp it used in the "name" attribute.

```js
const { data: therapiesList, isLoading: therapiesLoading } = useGetList(Resources.therapy);
```

You think it's over-engineering?)

Reply:
Ok, up to you. It's 🟢 type of comment)

Reply:
Anyway, I'd like to hear your opinion on this. If it doesn't make much sense, then I'll remove it.